src/DynamoCore/DSEngine/NodeToCode.cs

```diff
@@ -738,6 +738,9 @@ public string GetNextName()
                         var inputVar = node.GetAstIdentifierForOutputIndex(i).Value;
                         outputMap[inputVar] = string.Empty;
                     }
+
+                    var previewVar = node.AstIdentifierForPreview.Value;
+                    outputMap[previewVar] = string.Empty;
```

Comment:
Two questions:

1. Does this apply also to `DSFunction` node that has only one output port?
2. The reason you can use `AstIdentifierForPreview`, is it because it happens to be the intermediate variable name that we need for conversion?